_posts/2025-03-20-switching-to-effver.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: 'Versioning Zarr with EffVer'
+description: We are updating the versioning policy for the Zarr python library.
+date: 2025-01-09
+categories: blog
+permalink: /versioning-with-effver/
+---
+
+Back in January we released [Zarr-Python 3](https://zarr.readthedocs.io/en/v3.0.0/), the first new version of the library since 2016. While working on this release, we found that Zarr-Python's versioning policy didn't quite fit the needs of the project as it stands today. So we are modifying that versioning policy to make it better suited to the needs of Zarr-Python developers and users.
```

Comment:
```suggestion
Back in January we released [Zarr-Python 3](https://zarr.readthedocs.io/en/v3.0.0/), the first new major version of the library since 2016. After making this release, we found that Zarr-Python's versioning policy didn't quite fit the needs of the project as it stands today. So we have modified that versioning policy to make it better suited to the needs of Zarr-Python developers and users.
```